### DISCUSSION TITLE
Unable to find species in gene family

### DISCUSSION THREAD

**Discussion by ghost** [hahnlab/CAFE5#200] [Q&A] [answered]
I am having issues with my files when attempting to run CAFE. I have tried everything from the other discussions, but keep getting the error about the species not found in gene family x. 
[tree.txt](https://github.com/user-attachments/files/18088958/tree.txt)
[updated_coding_file.txt](https://github.com/user-attachments/files/18088959/updated_coding_file.txt)

**Comment by benfulton** [comment, marked as answer]
The first field in your species file should be a description, and you need to have a field in the header that says "ID" or something similar. So the first couple of lines should look like

```
Desc	Family ID	Alligator_mississippiensis	Apalone_spinifera	Carettochelys_insculpta	Chelonia_mydas	Chelonoidis_abingdonii	Chelydra_serpentina	Cuora_amboinensis	Cuora_mccordi	Dermatemys_mawii	Dermochelys_coriacea	Emydura_subglobosa	Gopherus_agassizii	Gopherus_evgoodei	Malaclemys_terrapin	Mesoclemmys_tuberculata	Pelodiscus_sinensis	Pelusios_castaneus	Platysternon_megacephalum	Podocnemis_expansa	Terrapene_carolina
(null)	OR1.3.7_CODING	7	11	0	1	6	3	4	6	1	1	3	3	2	7	4	8	5	0	3	6
(null)	OR10_CODING	176	342	282	146	215	309	292	337	272	75	516	416	488	326	330	369	191	340	285	331

```

Also I think your header line is space-separated even though the rest of the file is properly tab-separated.

The tree file needs to be on a single line, and it's missing a branch length for the top level (between the final two parenthesis).
